app/controllers/claims/support/schools/add_school_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Claims::Support::Schools::AddSchoolController < Claims::Support::ApplicationController
+  include WizardController
+
+  before_action :authorize_school
+  before_action :set_wizard
+
+  def update
+    if !@wizard.save_step
```

Comment:
```suggestion
    unless @wizard.save_step
```

Does this improve readability?

Reply:
This is in every wizard. I would be reluctant to change this, as it seems to be the accepted format.